### ISSUE TITLE
Input already masked images instead of generating patches

### ISSUE THREAD

**Issue by cryptedp** [pathak22/context-encoder#16]
Hi
Is it possible to input already masked images instead of generating masks in the image first and inpaint them ? The problem is, that the masks in my images will not all be in the same relative locations so I should inpaint different image regions but I don't want to generate masks myself on the image. If it is possible, should the masks satisfy some constraints (like pixel values )?
Thx in advance

**Comment by maryam089** [comment]
Did you find the solution to the above problem ? @cryptedp

**Comment by pathak22** [comment]
Yes, you can always modify the input code to do that. I only provided automatic masks for the convenience of code release and to avoid having to release masks file.

**Comment by niloleart** [comment]
Did you find the way to do that @cryptedp ?
It will help me a lot :)

**Comment by cryptedp** [comment]
Yes, it was quite a hack actually. We detected the objects and modified the image, such that the object is in the middle and then we called inpaint_center to inpaint that region. Next we took the inpainted region back to its original position. It did not work that bad, however if we had multiple object instances it was quite messy and sometime the inpainting result was not that good

**Comment by niloleart** [comment]
Can you provide me more info about it? Or the code? :D
Thanks, @cryptedp
